### PR TITLE
Guidelines for Community Engagement

### PR DESCRIPTION
**Pull request by it33**
Starting point for adding guidelines on engaging with community. Ready for review.

### CODE REVIEW

File: source/guides/community.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+To keep Mattermost “obvious” to users, please keep community discussion, documentation and on-screen help simple, effective and ready-to-translate.
+
+**Simple** 
+
+- Aim for the reading ability of an 11-year old. Use short sentences. Avoid large words.
+
+**Effective**
+
+- Start with what’s important, and leave less important details to the end, or omit them
```
Comment (lfbrock) [inline source/guides/community.rst]: Bulleted lists points should be consistent with periods

File: source/guides/community.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+That documentation machine-translated into German and then back into English looks like this:
+
+  There are some configuration settings you could know if your instance Matter Most violin
+
+The sentence would be more ready-to-translate to say “There are a few configuration settings you can adjust when setting up your instance of Mattermost.”
+
+Testing our documentation:
+------------------------------------------------
+
+One “trick” to use to review content is to ask: “How would Agent Coulson explain this concept to Thor? [1]”
```
Comment (lfbrock) [inline source/guides/community.rst]: Does this count as a culture-centric reference? Or do the avengers transcend any one culture. 

Also if the [1] is a reference I think you're missing the footnote

**Comment by it33** [comment]
@lfbrock updated

File: source/guides/community.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+1. Don't offer advice on database modification without a clear warning
+----------------------------------------------------------------------------
+
+Never give DB advice without a warning similar to: `Manual database modification bypass checks in the code and could make a system unrecoverable. It's not recommended and if you do make a change, make sure to back up your DB, and verify changes carefully.`
+
+2. Answer with docs
+----------------------------------------------------------------------------
+
+When answering a common question, always include a link to the docs so the person asking--along with people in future who read your response. If no doc exists, do one of two things:
```
Comment (lfbrock) [inline source/guides/community.rst]: I think this sentence is incomplete

File: source/guides/community.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+----------------------------------------------------------------------------
+
+Never give DB advice without a warning similar to: `Manual database modification bypass checks in the code and could make a system unrecoverable. It's not recommended and if you do make a change, make sure to back up your DB, and verify changes carefully.`
+
+2. Answer with docs
+----------------------------------------------------------------------------
+
+When answering a common question, always include a link to the docs so the person asking--along with people in future who read your response. If no doc exists, do one of two things: 
+
+1. Add the  the documentation, then answer the question with a link to the information, or
```
Comment (lfbrock) [inline source/guides/community.rst]: Two "the"s

File: source/guides/community.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+Below is a list of guidelines for maintainers, key contributors and Mattermost, Inc. staff to use when engaging with the broader community in forums, GitHub issues, Twitter, our Mattermost servers and other places of online conversation. 
+
+Best Practices 
+============================================================================
+
+1. Don't offer advice on database modification without a clear warning
+----------------------------------------------------------------------------
+
+Never give DB advice without a warning similar to: `Manual database modification bypass checks in the code and could make a system unrecoverable. It's not recommended and if you do make a change, make sure to back up your DB, and verify changes carefully.`
```
Comment (lfbrock) [inline source/guides/community.rst]: Is the text here supposed to be in-line code? If so I think it's two ticks for rst docs. Also this sentence is incorrect (should be modifications bypass or modification bypasses)

File: source/guides/community.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+2. Answer with docs
+----------------------------------------------------------------------------
+
+When answering a common question, always include a link to the docs so the person asking--along with people in future who read your response. If no doc exists, do one of two things: 
+
+1. Add the  the documentation, then answer the question with a link to the information, or 
+2. Answer the question and open a ticket for the missing information to be added to docs.
+
+During the triage process, if the information missed should have obviously been included, assign the ticket to the earliest approver for the doc PR, otherwise assign to doc PR author.
```
Comment (lfbrock) [inline source/guides/community.rst]: This is a little unclear, won't most docs we link to be pretty old, and may potentially have had multiple PRs made to them? Who counts as the doc PR approver? Seems complex for triage

**Comment by lfbrock** [comment]
+1

**Comment by yangchen1** [comment]
+1

**Pull request merged**